types/koa-bodyparser/index.d.ts

```diff
@@ -21,7 +22,7 @@ import * as Koa from "koa";
 
 declare module "koa" {
     interface Request {
-        body?: any;
+        body: unknown;
```

Comment:
this should remain as an optional field

Reply:
Actually according to the `koa-bodyparser` if body is empty, it's still an empty hash:

> // the parsed body will store in ctx.request.body
>  // if nothing was parsed, body will be an empty object {}